build-in-container/scripts/build.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+set -euo pipefail
+
+SOURCE_FOLDER="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
+
+CBL_MARINER_GIT_URL="https://github.com/microsoft/CBL-Mariner.git"
+
+VERBOSE=1
+LOG_LEVEL=info
+CCACHE_DIR=$BUILD_OUT_BASE_DIR/ccache
+SPECS_DIR=$SOURCE_FOLDER/SPECS
+USE_CCACHE="y"
+ARTIFACT_PUBLISH_DIR=""
+LOG_PUBLISH_DIR=""
+ERRORS_OCCURRED=0
+
+readonly MARINER_RELEASE_TAG="2.0-stable"
+
+if [ -t 1 ]; then
+    CYAN="\e[36m"
+    RESET="\e[0m"
+else
+    CYAN=""
+    RESET=""
+fi
+
+function log() {
+    timestamp="$(date "+%F %R:%S")"
+    echo -e "${CYAN}+++ $timestamp $1${RESET}"
+}
+
+function cleanup() {
+    log "Cleaning up..."
+
+    if [[ -n $LOG_PUBLISH_DIR ]]; then
+        publish_build_logs
+    fi
+}
+
+function download_mariner_toolkit() {
```

Comment:
While having this optimized for pipelines is good, we really need it to also support local developers. This should either use the local files, or be something that can be invoked easily.